src/common/Navigation/styled.js

```diff
@@ -11,31 +11,27 @@ export const BlackBar = styled.div`
 
 export const NavbarContainer = styled.nav`
     width: 1368px;
-    display: flex;
-    justify-content: space-between;
+    display: grid;
+    grid-template-columns: 1fr 1fr 2.5fr;
```

Comment:
1fr 1fr 2.5fr is the same as 2fr 2fr 5fr. It is all about ratio, this means that first and second column takes 2/9 of screen width, while last one takes 5/9 of screen width. I have changed it from 1fr 1fr 3fr where last column takes 3/5 screen width because it was too wide. 1fr 1fr 2.5fr resembles UI project more.

Here are some [key concepts of grid including fr unit](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout)